### PR TITLE
DDF-2155: added documentation for managing DDF-registry.

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Initial Draft of managing DDF Registry documentation.

#### Who is reviewing it?

@mcalcote @brianfelix @vinamartin @gordocanchola @clockard @oscaritoro 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@shaundmorris
@stustison
#### How should this be tested?

Documentation renders correctly and is free of typos.

#### Any background context you want to provide?

Details may change as registry is finalized.

#### What are the relevant tickets?

https://codice.atlassian.net/browse/DDF-2155
#### Screenshots (if appropriate)
na
#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by shaundmorris** [comment]
@ricklarsen Does this depend on other PRs being merged or does it make sense to merge the initial draft?

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc [hunk 1]
```diff
@@ -1,26 +1,530 @@
...
+|Entries are White List
+|whiteList
+|Boolean
+|A flag indicating whether or not the Registry Entry Ids represent a 'white list' (allowed - checked) or a 'black list' (blocked - unchecked) ids
+|
+|no
+
+|Registry Entries Ids
+|registryEntryIds
+|String0
```
Comment (vinamartin) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: Is there a reason for the 0? Not sure if typo or actual format....

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: typo.

**Comment by ricklarsen** [comment]
@shaundmorris It's not really dependent on any specific PR, but I know there are things that may still be moved/changed/renamed, so there's no reason to hurry this one. I just wanted to get it in front of people so we can get feedback.

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc [hunk 2]
```diff
@@ -1,26 +1,530 @@
...
+|Associations
+a|click *Add* button (icon:plus[]) to add custom fields
+|Configurable
+|no
+
+|===
+
+===== [[adding_a_service_binding]]Adding a Service Binding
+
+* Navigate to *Local Registry Nodes* tab under *${ddf-registry}* or *${ddf-catalog}*.
```
Comment (stustison) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: Probably need to tell them to go to the Admin Console first.

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: fixed.

**Comment by Lambeaux** [comment]
Do we want to allow this PR to go stale? Would it be better to get this in and have another iteration?

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc [hunk 3]
```diff
@@ -1,26 +1,532 @@
...
+
+${ddf-registry} contains the base registry components, plugins, sources and interfaces needed for ${branding} to function as a registry connecting multiple nodes.
 
 === ${ddf-registry} Prerequisites
 
 To use the ${ddf-registry}, the following apps/features must be installed:
 
-// a list
-* ${ddf-spatial}
+* <<Managing ${ddf-spatial},${ddf-spatial}>>
```
Comment (Lambeaux) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: This line seems out of place.

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc [hunk 4]
```diff
@@ -1,26 +1,532 @@
...
+|Property
+|Type
+|Description
+|Default Value
+|Required
+
+|Registry CUD Attributes
+|writeAccessPolicyStrings
+|String
+|Roles/attributes required for CUD (create/update/delete) operations on registry entries. Example: {role=role1;type=type1}
```
Comment (Lambeaux) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: Do we want to use the more familiar acronym `CRUD` for create/read/update/delete operations? Or does reading have no place here?

Comment (Lambeaux) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: I see the read attributes below. Ignore my comment.

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc [hunk 5]
```diff
@@ -1,26 +1,532 @@
...
+|yes
+
+|Security Attributes
+|Security attributes associated with this node.
+|String
+|
+
+|Last Updated
+|Date this entry's data was last updated
+|
```
Comment (Lambeaux) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: Missing value.

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc [hunk 6]
```diff
@@ -1,26 +1,532 @@
...
+|
+
+|Live Date
+|Date indicating when this node went live or operational
+|
```
Comment (Lambeaux) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: Missing value.

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc [hunk 7]
```diff
@@ -1,26 +1,532 @@
...
+|string
+|
+
+|Service Version
+|This service version
+|string
+|
+
+|Service Type
+|Identifies the type of service this is by a urn
```
Comment (Lambeaux) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: `urn` doesn't seem right ... should it be an acronym? Like `URN`? Is it a spelling error?

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc [hunk 8]
```diff
@@ -1,26 +1,532 @@
...
+|Required
+
+|Contact Title
+|Contact Title
+|String
+|yes
+
+|Contact First Name
+|Contact First Name
+|String|yes
```
Comment (Lambeaux) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: Bring `|yes` down to a new line.

**Comment by Lambeaux** [comment]
Couple of comments to address. Then :shipit: !!

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc (hunk 3 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: it's just redone to include a link to the spatial section of the managing docs.

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc (hunk 4 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: done

**Comment by shaundmorris** [comment]
@Lambeaux as long as the docs are accurate with the code/functionality I'm a fan of merging and doing another round of updates when the docs need to be synced up.

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc (hunk 7 above)
Comment (vinamartin) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: https://en.wikipedia.org/wiki/Uniform_Resource_Name

**Comment by ricklarsen** [comment]
I'll do one more pass through then and we can get someone to confirm it builds.

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc (hunk 7 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: Looks like it's usually capitalized (outside of wikipedia)

Comment (vinamartin) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: I feel like it looks fine either way. 👍  Whatever makes sense for other people.

**Comment by stustison** [comment]
👍

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc (hunk 8 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: fixed.

**Comment by ricklarsen** [comment]
Should be all updated now

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc [hunk 9]
```diff
@@ -1,26 +1,533 @@
...
+|string
+|yes
+
+|Address
+|This organization's primary address
+|Expand to enter address information
+|yes
+
+|TelephoneNumber
+|||
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: Are telephone number and email missing descriptions?

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc [hunk 10]
```diff
@@ -1,26 +1,533 @@
...
+* Click Save.
 
 ==== Publishing to Others
 
-===== Editing Publishing Settings
+* Navigate to the *Remote Registries* tab in either ${ddf-registry} or ${ddf-catalog} application.
+* Click *Add* to add a remote registry.
+* Give node a unique local name.
+* Enter Registry Service (CSW) Url.
+* Confirm Allow Publish is checked.
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: The UI has "allow push" currently, is it going to be changed to "allow publish"?

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc [hunk 11]
```diff
@@ -1,26 +1,533 @@
...
+* Under *Operations*, click *Publish to ... * link.
+
+==== Subscribing to Another Node
+
+* Navigate to the *Remote Registries* tab in either ${ddf-registry} or ${ddf-catalog} application.
+* Click *Add* to add a remote registry.
+* Give new node a unique Local name.
+* Add the URL to access node.
+* Enter any needed credentials in the Username/password fields.
+* Click *Save*/*Add*.
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: So by adding a remote registry, DDF automatically subscribes to it?

**Comment by shaundmorris** [comment]
👍  after last comments are addressed/resolved. Verified build, documentation steps, and reviewed PDF/HTML formats.

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc (hunk 9 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: yes. fixed.

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc (hunk 10 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: I'm not sure. @mcalcote ?

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc (hunk 11 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: I thought that was the case, but I'll confirm with @mcalcote

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc (hunk 10 above)
Comment (mcalcote) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: I was under the impression that we were sticking with the Push/Pull terminology, but I also know that some of those details have been in discussion recently. For now it is "Push".

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc (hunk 11 above)
Comment (mcalcote) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: The current functionality is that it does automatically subscribe.

File: distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc (hunk 10 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc]: fixed.

**Comment by ricklarsen** [comment]
All comments addressed

**Pull request merged**